packages/shared/src/h5t.ts

```diff
@@ -0,0 +1,53 @@
+/* ------------------- */
+/* ---- H5T ENUMS ---- */
+
+// https://docs.hdfgroup.org/hdf5/v1_14/_h5_tpublic_8h.html#title3
+
+export enum H5T_CLASS {
+  INTEGER = 0,
+  FLOAT = 1,
+  TIME = 2,
+  STRING = 3,
+  BITFIELD = 4,
+  OPAQUE = 5,
+  COMPOUND = 6,
+  REFERENCE = 7,
+  ENUM = 8,
+  VLEN = 9,
+  ARRAY = 10,
+}
+
+export enum H5T_ORDER {
+  LE = 0,
+  BE = 1,
+  VAX = 2,
+  MIXED = 3,
+  NONE = 4,
+}
+
+export enum H5T_SIGN {
+  NONE = 0, // unsigned
+  SIGN_2 = 1, // signed
+  NSGN = 2,
```

Comment:
Not sure what this enum member represents, TBH.

Reply:
Even the HDF people don't know apparently: https://docs.hdfgroup.org/hdf5/v1_14/_h5_tpublic_8h.html#af7bfee2db210a12b9290eba85d730a71

Perhaps @t20100 has an idea ?